### PR TITLE
Hide disconnected idps when the flag is true

### PR DESCRIPTION
**Pull request by Badlapje**
Hide unconnected idps when the flag says they are not desired

### CODE REVIEW

File: theme/README.md [hunk 1]
```diff
@@ -143,3 +143,15 @@ Below you'll find a list of the "entry points" for each page with corresponding
...
+- rememberChoiceFeature: allow users to save their selected IdP and then auto-select it on returning visits.  You can test this by going to `/functional-testing/wayf?rememberChoiceFeature=1`.
+- backLink: display a link which, when clicked, allows you to go back two pages.  In essence the equivalent of clicking the back button twice.  You can test this by going to `/functional-testing/wayf?backLink=1`.
+- cutoffPointForShowingUnfilteredIdps: unlike the previous flags which were booleans, this one is an integer.  When this flag is present & the user is not searching (so no value in the search field): there should only be idps shown when there are no more than the cutoff point.  So as an example: if there are 100 idps that would be shown normally, and the cutoff point is 50: the user should see no idps at all (so only the empty search field is shown).  You can test this by going to `/functional-testing/wayf?cutoffPointForShowingUnfilteredIdps=50`.
+- showConsentExplanation: whether or not to show a special explanation on the consent page.  There is currently no testpoint for this.
```
Comment (MKodde) [inline theme/README.md:157]: Nice addition. But I think you misinterpreted my idea to add a description of these parameters to a readme file. I was talking about the `parameters.yml` EngineBlock feature flags.

These parameters described above are already in this readme: https://github.com/OpenConext/OpenConext-engineblock/blob/master/docs/js_testing.md

let's leave this as-is for now as this section demonstrates the available parameters in a more user friendly manner. Let's merge things in the 'leftover' ticket at the end of this development effort. Same goes for documenting the parameters.yml feature flags

File: theme/base/templates/modules/Authentication/View/Proxy/Partials/WAYF/idp/idpList.html.twig [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
-            {% include '@theme/Authentication/View/Proxy/Partials/WAYF/idp/idp.html.twig' with { idp: idp } %}
-        </li>
+        {% if not showRequestAccess and idp['connected'] is defined and not idp['connected'] %}
+            {# do nothing #}
+        {% else %}
+            <li>
+                {% include '@theme/Authentication/View/Proxy/Partials/WAYF/idp/idp.html.twig' with { idp: idp }
+                %}
+            </li>
+        {% endif %}
```
Comment (MKodde) [inline theme/base/templates/modules/Authentication/View/Proxy/Partials/WAYF/idp/idpList.html.twig:10]: Please rewrite this into a single blocked control statement. A single if without the else block should be possible here.

File: theme/cypress/integration/skeune/wayf/WayfGeneralBehaviour.js [hunk 3]
```diff
@@ -26,6 +26,19 @@ context('WAYF behaviour not tied to mouse / keyboard navigation', () => {
...
+      cy.get('.wayf__idp--noAccess')
+        .should('have.length', 5)
+        .should('be.visible');
+    });
+
+    it.only('Should show no disconnected IdPs when the flag is false', () => {
+      cy.visit('https://engine.vm.openconext.org/functional-testing/wayf?displayUnconnectedIdpsWayf=0&unconnectedIdps=5');
+      cy.get('.wayf__idp--noAccess')
+        .should('not.exist');
+    });
```
Comment (MKodde) [inline theme/cypress/integration/skeune/wayf/WayfGeneralBehaviour.js:41]: :muscle:

**Review by MKodde — CHANGES_REQUESTED** [review]
Fine work! See some feedback below.

File: theme/base/templates/modules/Authentication/View/Proxy/Partials/WAYF/idp/idpList.html.twig (hunk 2 above)
Comment (Badlapje) [reply theme/base/templates/modules/Authentication/View/Proxy/Partials/WAYF/idp/idpList.html.twig]: fixed

**Review by MKodde — APPROVED** [review]

**Pull request merged**